service/controller/app/v1/resource/status/create_test.go

```diff
@@ -27,7 +27,7 @@ func Test_Resource_EnsureCreated(t *testing.T) {
 			name: "case 0: update status flow",
 			obj: &v1alpha1.App{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:      "my-cool-prometheus",
+					Name:      "prometheus",
```

Comment:
Test was failing because the CR name was different. Now `prometheus` is always used.